infra/cifuzz/cifuzz.py

```diff
@@ -427,8 +427,9 @@ def run_fuzzers(  # pylint: disable=too-many-arguments,too-many-locals
     if not testcase or not stacktrace:
       logging.info('Fuzzer %s, finished running.', target.target_name)
     else:
-      logging.info(b'Fuzzer %s, detected error: %s.', target.target_name,
-                   stacktrace)
+      utils.print(b'Fuzzer %s, detected error: %s.' % (
```

Comment:
should we also have a newline after the "detected error:" part?

Reply:
Done.